mcs/class/System/common_networking.sources

```diff
@@ -21,7 +21,6 @@ System.Net/ChunkedInputStream.cs
 System.Net/EndPointListener.cs
 System.Net/EndPointManager.cs
 
-System.Net/FtpRequestCreator.cs
```

Comment:
The file can be deleted too.

I also just saw we have FtpRequestCreator.platformnotsupported.cs for WatchOS, that will need adjustments too (probably by moving the code into the corefx FtpWebRequest.cs)